### ISSUE TITLE
Conveying identity 'trust status' indicator (transferred from WP2)

### ISSUE THREAD

**Issue by rebecca-copeland** [reTHINK-project/governance-security-implementation#5]
During WP4 confcall, Ingo and I raised the idea of conveying identity 'Trust Status' indicator, which can be displayed to users, according to various rules of data protection. This brings some comfort feel to users before they click on address-book entry, destinations confirmed by authentication, incoming connectivity service request etc. 
The Trust Status indicator should show current status according to available information. If it is not just 'green' light, then the user may wish to update the information (a good incentive to do so), or make it more specific (e.g. remove the Mickey Mouse picture)...
The idea is that Trust Status will be conveyed, but it is up to the particular application to display it or not, or to display only 'green' or '?' etc.
We should try to define what are the Trust Status classes, who define them and how are they expressed and conveyed. 
Is this going beyond the existing security token? Is this an extension of it? Can CSPs add to it from their own knowledge?

**Comment by ingofriese** [comment]
I agree and I strongly support this point - to have some kind of trust indicator -.
But I'm a bit reluctant with abstract Trust Status classes, because the messurement of trust is quite individuel and should be up to the CSP or even better up to the user.
I'd like to see some more detailed Trust Classes....like for Example
a call comes in and a trust engine/service whatever....says
1)....the  claimed identiy "ingo.friese@telekom.de" in the offer message is really proved by telekom.de
because the digital signature is checked. The Id is not faked.
2) ....the user has alreay talked before to this ID
3).....others in the users social graph have  talked to this ID before
4)....the ID is not on a blacklist...its on a white list
etc

And in relation on 1,2,3,4 checks any client/CSP/user can build his own trust indicator as a green/yellow/grey symbol
